### ISSUE TITLE
SpriteFrames editor window: increase Animation Speed FPS arrow change from 0.01 to 1

### ISSUE THREAD

**Issue by hsandt** [godotengine/godot-proposals#6101]
### Describe the project you are working on

Platformer with character animations via spritesheet

### Describe the problem or limitation you are having in your project

Modifying the Speed (FPS) of animation in SpriteFrames with the up/down arrows is too slow, as it modifies the value by increments of 0.01.

![image](https://user-images.githubusercontent.com/1067772/212484987-d2d982fe-b0bd-4d81-ae40-194fc119ae86.png)

After one press on the up arrow:

![image](https://user-images.githubusercontent.com/1067772/212485034-a5ccbb72-d20d-47cb-962a-f3d17f48e67d.png)

After holding press for 3 seconds:

![image](https://user-images.githubusercontent.com/1067772/212485108-7c6482b0-4755-4e61-a475-8a0034963c8e.png)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Main suggestion:

We could default the increment/decrement to +1/-1 FPS.

Bonus:

* We could keep the precise increment/decrement (0.1 or 0.01) when holding Ctrl or Shift when clicking on the arrow.
* We could add alternative keyboard inputs like press Up/Down with modifiers to increment/decrement by 1, 0.1, 0.01.

I think the main suggestion is key, because most animators will think in terms of integer FPS, esp. standards like 24 FPS, 30 FPS and 60 FPS. So far I've only ever needed integer FPS, it was precise enough to define all my animations. But the bonus suggestions, as well as manual value input, would allow the user to still input precise values if needed.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

![image](https://user-images.githubusercontent.com/1067772/212485278-722fd460-0a19-4984-92c8-2ab62a2a899a.png)

Press up arrow:

![image](https://user-images.githubusercontent.com/1067772/212485293-18d2d426-2e07-4ab3-8195-3f2875be5e5e.png)

When holding, it will also increase the FPS 1 by 1. However, this would be quite fast with the current increment *repeat* period which was tuned for 0.01, so I imagine that we should reduce the repeat period to around 6 FPS per second held.

And you can imagine the same with down arrow and decrement.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No, it's core editor UI.

### Is there a reason why this should be core and not an add-on in the asset library?

Same, it's core editor UI.

**Comment by dalexeev** [comment]
Yes, this is possible via `SpinBox.custom_arrow_step` (but we shouldn't change `Range.step`). The same applies to the new Frame Duration field.

We also need to correct the position of labels for empty frames in the SpriteFrames editor:

![](https://user-images.githubusercontent.com/47700418/212486757-6e7ebaca-14fc-4802-b4ee-815f465dab6a.png)

**Comment by hsandt** [comment]
A developer tackling this issue may not to have a look at this bug: https://github.com/godotengine/godot/issues/71419 since it may affect similar areas in the code (if only to understand merge conflicts if another dev handles it at the same time).

**Comment by Calinou** [comment]
Now that https://github.com/godotengine/godot-proposals/issues/226 is implemented, we can make the buttons increase/decrease the value by `1` while still allowing manual values to have `0.01` precision.

**Comment by RedMser** [comment]
A similar problem happens when dragging on the custom minimum size of Controls. It's measured in pixels but it goes up by 0.01 steps.

**Comment by Calinou** [comment]
> A similar problem happens when dragging on the custom minimum size of Controls. It's measured in pixels but it goes up by 0.01 steps.

This will require adding a new value to the `PROPERTY_HINT_RANGE` property hint (possibly an optional 4th unnamed parameter, e.g. `0,10000,0.01,1`).
